website/docs/r/log_analytics.html.markdown

```diff
@@ -0,0 +1,64 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_log_analytics"
+sidebar_current: "docs-azurerm-resource-log-analytics"
+description: |-
+  Creates a new Log Analytics (formally Operational Insights Workspaces) Workspace.
+---
+
+# azurerm\_log\_analytics
+
+Creates a new Log Analytics (formally Operational Insights Workspaces) Workspace.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-01"
+  location = "East US"
+}
+
+resource "azurerm_log_analytics" "test" {
+  name                = "acctest-01"
+  location            = "${azurerm_resource_group.test.location}"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+  sku                 = "Standard"
+  retention_in_days   = 30
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Specifies the name of the Workspace. Changing this forces a new resource to be created. Workspace name should include 4-63 letters, digits or '-'. The '-' shouldn't be the first or the last symbol.
```

Comment:
can we update this so that `Changing this forces a new resource to be created.` is the last sentence here?

Reply:
Done!